Dockerfile.run

```diff
@@ -1,13 +1,15 @@
+FROM alpine:3.4
 
-FROM    alpine:3.4
-RUN     apk -U add \
-            python \
-            py-pip
 
-COPY    requirements.txt /code/requirements.txt
-RUN     pip install -r /code/requirements.txt
+COPY dist/docker-compose-Linux-x86_64 /usr/local/bin/docker-compose
 
-ADD     dist/docker-compose-release.tar.gz /code/docker-compose
-RUN     pip install --no-deps /code/docker-compose/docker-compose-*
+RUN apk update && apk add --no-cache \
+    curl openssl ca-certificates \
+    && chmod +x /usr/local/bin/docker-compose \
+    && wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-glibc/master/sgerrand.rsa.pub \
+    && wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.23-r3/glibc-2.23-r3.apk \
+    && apk add --no-cache glibc-2.23-r3.apk && rm glibc-2.23-r3.apk \
```

Comment:
Please move the version here to an env variable:

```
export GLIBC=2.23-r3;
```

It can be used in the `wget` and `apk add` lines. That way when we upgrade the version we only need to change it in one place, not 4.